src/lib.rs

```diff
@@ -1,2 +1,16 @@
 pub use fil_actors_runtime as runtime;
+pub use interface_trait::StructSignature;
 pub use primitives;
+
+#[test]
+fn demo_struct_signature_derive() {
+    #[derive(StructSignature)]
+    #[allow(dead_code)]
+    struct Foo {
+        pub test: String,
+        pub test2: String,
+        pub test3: Option<String>,
+    }
+
+    println!("{:?}", Foo::signature());
```

Comment:
What about adding an `assert_eq!` to validate that the hash computation is correct?